modules/exploits/unix/http/cacti_filter_sqli_rce.rb

```diff
@@ -0,0 +1,224 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'metasploit/framework/hashes/identify'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Cacti color filter authenticated SQLi to RCE',
+        'Description' => %q{
+          This module exploits a SQL injection vulnerability in Cacti 1.2.12 and before. An admin can exploit the filter
+          variable within color.php to pull arbitrary values as well as conduct stacked queries.  With stacked queries, the
+          path_php_binary value is changed within the settings table to a payload, and an update is called to execute the payload.
+          After calling the payload, the value is reset.
+        },
+        'License' => MSF_LICENSE,
+        'Author' =>
+          [
+            'h00die', # msf module
+            'Leonardo Paiva', # edb, RCE
+            'Mayfly277' # original github, M4yFly on twitter SQLi
+          ],
+        'References' =>
+          [
+            [ 'EDB', '49810' ],
+            [ 'URL', 'https://github.commandcom/Cacti/cacti/issues/3622' ],
+            [ 'CVE', '2020-14295' ]
+          ],
+        'Platform' => ['unix'],
+        'Privileged' => false,
+        'DefaultOptions' => {
+          'PrependFork' => true
```

Comment:
?

Reply:
early attempt to not have to create a new login session for cleanup... but alas, didn't come true.